permissions/plugin-cloudbees-bitbucket-branch-source.yml

```diff
@@ -4,9 +4,10 @@ github: "jenkinsci/bitbucket-branch-source-plugin"
 paths:
 - "org/jenkins-ci/plugins/cloudbees-bitbucket-branch-source"
 developers:
-- "amuniz"
-- "stephenconnolly"
 - "abayer"
-- "svanoort"
-- "rsandell"
+- "amuniz"
+- "bitwiseman"
+- "carroll"
+- "dnusbaum"
 - "jetersen"
```

Comment:
Feel free to remove me 😓 I do not plan on using bitbucket nor do I have the time to partake in the plugin maintenance 
```suggestion
```